pages/02.reference/01.chat/04.events/12.SkillAttribution/doc.md

```diff
@@ -14,6 +14,7 @@ Sent from the server when a skill execution has been triggered by a user.
     "type": "event",
     "event": "SkillAttribution",
     "data": {
+        "id": "6e291af8-fd5d-4c9a-8303-b2232cba294d",
```

Comment:
Should this be a UUIDv1?

Reply:
It should be. I'll update the sample since this is only a guid :)

Reply:
Fixed